.github/workflows/test.yml

```diff
@@ -38,8 +38,9 @@ jobs:
       - uses: ./.github/actions/cache-on-main
         with:
           path: node_modules/.cache
-          key: ${{ runner.os }}-tsc-${{ github.run_id }}
-          restore-keys: ${{ runner.os }}-tsc-
+          # Types must be regenerated if any typings change: dependencies or the GraphQL schema.
+          key: ${{ runner.os }}-tsc-${{ hashFiles('yarn.lock', 'src/graphql/**/schema.graphql') }}-${{ github.run_id }}
```

Comment:
do we need to check in the `schema.graphql` files for this to run?

https://github.com/Uniswap/interface/blob/main/.gitignore#L11

Reply:
i just figured this got created during the build process

Reply:
no, it will exist because it has been downloaded as part of `./.github/actions/setup` on L37.